DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ifupdown2/conftest.py

```diff
@@ -0,0 +1,57 @@
+import pytest_asyncio
+import pytest
+
+from dent_os_testbed.lib.interfaces.interface import Interface
+
+from dent_os_testbed.utils.test_utils.tgen_utils import tgen_utils_get_dent_devices_with_tgen
+
+from dent_os_testbed.test.test_suite.functional.ifupdown2.ifupdown2_utils import (
+    IFUPDOWN_CONF, IFUPDOWN_BACKUP,
+    INTERFACES_FILE,
+)
+
+
+async def _apply_config(dent_dev, fields, config_file=IFUPDOWN_CONF):
+    # Prep env and change it
+    opts = [f's~^{k}=.*~{k}={v}~' for k, v in fields.items()]
+    rc, _ = await dent_dev.run_cmd(f'sed -i -E "{"; ".join(opts)}" {config_file}')
+    return rc
+
+
+async def _copy(dent_dev, src, dst, do_assert=True):
+    rc, out = await dent_dev.run_cmd(f'cp {src} {dst}', sudo=True)
+    if do_assert:
+        assert not rc, f'Failed to copy.\n{out}'
+
+
+@pytest_asyncio.fixture()
+async def prepare_env(testbed):
```

Comment:
This name is too abstract. Need to thing of a better one.

Reply:
Done